src/lib/y2users/linux/base_reader.rb

```diff
@@ -42,9 +42,6 @@ def read
         Config.new.tap do |config|
           read_elements(config)
           read_passwords(config)
-          read_authorized_keys(config)
```

Comment:
Please, remove the unneeded requires (i.e, login_config, useradd_config_reader and ssh_authorized_keyring).

Reply:
Arrrrgghhh! :man_facepalming: :man_facepalming: :man_facepalming: